### ISSUE TITLE
assert fails on example in the docs

### ISSUE THREAD

**Issue by stringertheory** [requests-cache/requests-cache#824]
Hi, thank you for your work on this project!

### The problem
In [this example](https://requests-cache.readthedocs.io/en/stable/user_guide/matching.html#cache-keys) in the docs, the assertion fails (and the second request is a cache miss) because `response_1.cache_key` is different from the other two:

```
session = CachedSession(ignored_parameters=['foo'])
response_1 = session.get('https://example.com')          # cache miss
response_2 = session.get('https://example.com?foo=bar')  # cache hit
response_3 = session.get('https://example.com?foo=qux')  # cache hit
assert response_1.cache_key == response_2.cache_key == response_3.cache_key
```

### Expected behavior
I don't have a strong point of view on whether this is the desired behavior, but was expecting for the behavior to match the documentation.

### Environment
- requests-cache version 1.0.1
- python version 3.11.1

**Comment by JWCook** [comment]
Thanks for catching that! Thankfully this is only an error in the docs, and not a bug. This behavior did change in 1.0 due to #668 and #694.

TL;DR:
* Previously, a request with an ignored parameter would be cached the same as if that parameter was missing.
* Currently, the presence of an ignored parameter (but not its value) is included in the cache key. So in this example, `response_1.cache_key` is expected to be different.

So the correct assertion now is:
```python
assert response_2.cache_key == response_3.cache_key
```

This has been fixed in main.
